rpm/stackdriver-agent.spec

```diff
@@ -276,13 +217,6 @@ rm -rf %{buildroot}%{_prefix}/include/curl %{buildroot}%{_prefix}/lib/libcurl*
 %{__rm} -f %{buildroot}%{_prefix}/man/man5/%{programprefix}collectd-snmp.5*
 %{__rm} -f %{buildroot}%{_prefix}/bin/stackdriver-utils_vl_lookup_test
 
-%if %{bundle_yajl}
-mkdir -p %{buildroot}%{_libdir}/yajl/
-cp /usr/lib64/libyajl.so.1 %{buildroot}%{_libdir}/yajl/
-ln -s -t %{buildroot}%{_libdir} yajl/libyajl.so.1
-cp /usr/share/doc/yajl-1.0.7/COPYING yajl.COPYING
-%endif
-
 %post
```

Comment:
Should we move the `systemctl daemon-reload` bit of the install script here?

Reply:
After conversation, let's put this off until the systemd refactor.

Reply:
So... @ThatDevopsGuy any update on that systemd refactor? :D 

Would be nice to have unit files on systems with systemd instead of relying on systemd-sysv-generator.